src/rz/angular/effects/service/service.ts

```diff
@@ -1,7 +1,13 @@
+import { EditFileEffect } from "../../../morph/interfaces/morph.interface";
 import { exportTsFiles, isTsFile } from "../../../utils/add-export";
+import { exportInIndex } from "../../export-in-index";
 
 export function exportServiceFile(filePathWithName: string): void {
     if (isTsFile(filePathWithName)) {
         exportTsFiles(filePathWithName);
     }
+}
+
+export function serviceEffects(fileEffects: EditFileEffect[]): EditFileEffect[]{
```

Comment:
We will write a unit test for service.ts to confirm working as expected. Will look similar to what is there for standalone components.